src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -25,4 +23,28 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
+
+    @GetMapping
+    public ResponseEntity getLines() {
```

Comment:
제네릭 타입을 설정해서 응답을 명확히 해주시면 좋을 것 같습니다 :)

```suggestion
    public ResponseEntity<LineResponse> getLines() {
```

Reply:
넵! 꼼꼼하게 신경 쓰도록 해보겠습니다 ㅎ